clients/unshaded/src/main/java/tachyon/client/TachyonFile.java

```diff
@@ -495,7 +495,7 @@ boolean recache(int blockIndex) throws IOException {
       inputStream.skip(offset);
 
       int bufferBytes =
-          (int) mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB) * 4;
+          (int) mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB);
```

Comment:
Was the original intention to default to 4MB? Is one buffer size more appropriate than the other?